### DISCUSSION TITLE
EDC 0.7.1 → /proxy → value.yaml

### DISCUSSION THREAD

**Discussion by jerzy-wachowiak** [eclipse-tractusx/tractusx-edc#1338] [Q&A]
To use this new API
```
POST /aas/request HTTP/1.1
Host: https://consumer-data.plane/proxy
X-Api-Key: password
Content-Type: application/json
```
`- proxy` should be added to https://github.com/eclipse-tractusx/tractusx-edc/blob/724b40d4dc6b411515c5474355a3529fba4924de/charts/tractusx-connector-azure-vault/values.yaml#L451-L452

Is this correct? Maybe it should be added as default to the Helm chart?

**Comment by wolf4ood** [comment]
Hi @jerzy-wachowiak 

i'm not sure it needs to be there. The proxy api is just an helper for fetching data, but imho it should not be exposed as the public API.

**Comment by jerzy-wachowiak** [comment]
https://github.com/eclipse-tractusx/tractusx-edc/blob/724b40d4dc6b411515c5474355a3529fba4924de/charts/tractusx-connector-azure-vault/values.yaml#L361-L364
The API is protected by a key. Is this not a similar idea as in the control plane at https://github.com/eclipse-tractusx/tractusx-edc/blob/724b40d4dc6b411515c5474355a3529fba4924de/charts/tractusx-connector-azure-vault/values.yaml#L229 ?
